core/src/main/java/org/elasticsearch/client/transport/TransportClientNodesService.java

```diff
@@ -360,9 +360,10 @@ protected void doSample() {
                     try {
                         // its a listed node, light connect to it...
                         logger.trace("connecting to listed node (light) [{}]", listedNode);
-                        transportService.connectToNodeLight(listedNode);
+                        transportService.connectToNodeLight(listedNode, pingTimeout, !ignoreClusterName);
```

Comment:
Actually it is only in that one case.

Though I wonder if we need a new action - why not just use the liveness action?